messages/device.login.json

```diff
@@ -7,5 +7,6 @@
   ],
   "actionRequired": "Action Required!",
   "enterCode": "Enter %s user code in the verification URL %s",
-  "success": "Login successful for %s. You can now close the browser."
+  "success": "Login successful for %s. You can now close the browser.",
+  "invalidInstanceUrl": "Invalid instance URL"
```

Comment:
This needs more. `The instance URL %s is invalid because it points to the lightning domain which can not be used for authentication with the CLI` or something like that. @jshackell-sfdc FYI.